lib/schedule.js

```diff
@@ -188,40 +188,53 @@ Job.prototype.schedule = function(spec) {
   var inv;
   var start;
   var end;
+  var timezone = null;
 
   if (typeof spec === 'object' && spec.rule) {
     start = spec.start || null;
     end = spec.end || null;
+    timezone = spec.timezone || null;
     spec = spec.rule;
 
     if (start != null) {
-      if (!(start instanceof Date)) {
-        start = new Date(start);
+      if (!(start instanceof CronDate)) {
+        start = new CronDate(start, timezone);
       }
       if (!isValidDate(start) || start.getTime() < Date.now()) {
         start = null;
       }
+
     }
 
-    if (end != null && !(end instanceof Date) && !isValidDate(end = new Date(end))) {
+    if (end != null &&
+      !(end instanceof Date) &&
+      !(end instanceof CronDate) &&
+      !isValidDate(end) &&
+      !isValidDate(end = new CronDate(end, timezone))) {
       end = null;
     }
   }
 
   try {
-    var res = cronParser.parseExpression(spec, { currentDate: start });
+
+    var res = cronParser.parseExpression(spec, { currentDate: start || undefined, tz: timezone });
     inv = scheduleNextRecurrence(res, self, start, end);
     if (inv !== null) {
       success = self.trackInvocation(inv);
     }
 
   } catch (err) {
     var type = typeof spec;
-    if ((type === 'string') || (type === 'number')) {
+    if (type === 'string') {
       spec = new Date(spec);
     }
```

Comment:
While you're here could you remove the `type` variable and use `if (typeof spec === 'string')`